deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,663 @@
+                image: rook/ceph:v1.0.1
+                name: rook-ceph-operator
+                resources: {}
+                volumeMounts:
+                - mountPath: /var/lib/rook
+                  name: rook-config
+                - mountPath: /etc/ceph
+                  name: default-config-dir
+              serviceAccountName: rook-ceph-system
+              volumes:
+              - emptyDir: {}
+                name: rook-config
+              - emptyDir: {}
+                name: default-config-dir
+
+        # NooBaa operator deployment
+      - name: noobaa-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              noobaa-operator: deployment
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: noobaa
+                noobaa-operator: deployment
+            spec:
+              containers:
+              - env:
+                - name: OPERATOR_NAME
+                  value: noobaa-operator
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                image: noobaa/noobaa-operator:1.0.2
+                imagePullPolicy: IfNotPresent
+                name: noobaa-operator
+                resources:
+                  limits:
+                    cpu: 250m
+                    memory: 256Mi
+              serviceAccountName: noobaa-operator
+
+      permissions:
+        # OCS operator permissions
+      - serviceAccountName: ocs-operator
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - ocs.openshift.io
+          resources:
+          - '*'
+          - storageclusters
+          - ocsinitializations
+          verbs:
+          - '*'
+        - apiGroups:
+          - ceph.rook.io
+          resources:
+          - cephclusters
+          verbs:
+          - create
+          - get
+          - delete
+          - update
+          - list
+
+        # Rook-Ceph permissions
+      - serviceAccountName: rook-ceph-system
```

Comment:
where are these RBAC permissions being sourced from for the rook-dev-* and noobaa operators? Do we have a way to keep these RBAC permissions in sync with the specific release of rook/noobaa operators that are being deployed in this CSV?

Reply:
These were sourced from the rook-ceph and noobaa operator CSV in the operator-framework/community-operators. In this PR I've manually merged them in. But we are working on automation to generate this, which will be coming in another PR.